package.json

```diff
@@ -27,10 +27,10 @@
     "@microsoft/eslint-formatter-sarif": "^3.1.0",
     "@playwright/experimental-ct-vue": "^1.49.1",
     "@playwright/test": "^1.49.1",
-    "@storybook/addon-essentials": "^8.4.7",
-    "@storybook/blocks": "^8.4.7",
-    "@storybook/vue3": "^8.4.7",
-    "@storybook/vue3-vite": "^8.4.7",
+    "@storybook/addon-essentials": "8.5.0-alpha.22",
+    "@storybook/blocks": "8.5.0-alpha.22",
+    "@storybook/vue3": "8.5.0-alpha.22",
+    "@storybook/vue3-vite": "8.5.0-alpha.22",
```

Comment:
We might want to consider moving all of these these version references to the workspace file so it's easier to maintain the versions.